src/main/java/org/openstreetmap/atlas/utilities/command/ReflectionUtilities.java

```diff
@@ -0,0 +1,88 @@
+package org.openstreetmap.atlas.utilities.command;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.AbstractAtlasShellToolsCommand;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.AtlasShellToolsMarkerInterface;
+
+import io.github.lukehutch.fastclasspathscanner.FastClasspathScanner;
```

Comment:
Sorry, you'll have to update this to the new ClassGraph dependency, it might actually make this code a bit simpler.

Reply:
I actually played with this initially, but didn't want to make the PR more complicated by adding dependency updates - so I'm glad you're bringing this change in.